src/edgepi/relay/README.md

```diff
@@ -0,0 +1,25 @@
+# EdgePi Relay Module
+Relay module is used to control the state of the relay. It can be setup to open/close the relay.
+
+# Hardware
+Relay_NO and Relay_COM on the terminal block can be used to setup a relay circuit.
+
+# Example Code
+```python
+
+
+
+```
+
+
+# Functionalities
+
+```python
+
+```
+
+# User Guide
+
+
+# Limitations 
+
```

Comment:
don't you want to fill in the example code and other sections?

Reply:
> don't you want to fill in the example code and other sections?

I was gonna fill the section after staging, when hardware test is done